core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -118,6 +120,13 @@ public static long getCurrentMs() {
     return System.currentTimeMillis();
   }
 
+  /**
+   * @return the current jvm major version, such as 1.8 or 11
```

Comment:
Is this comment true? Will it be `1.8` or will it be `8`?

Reply:
fixed comment